roles/mailserver/files/docker-compose.yml

```diff
@@ -126,8 +126,37 @@ services:
       - OPENCART_HOST=store.{{ domain_name }}
       - OPENCART_USERNAME=opencart
       - OPENCART_PASSWORD=changeThis
-      - SMTP_HOST=mailserver
-      - SMTP_PORT=25
+      - SMTP_HOST=mailserver 
+      - SMTP_PORT=25 
+      - SMTP_USER=username@domain.ext
+      - SMTP_PASSWORD=password
+{% endif %}
```

Comment:
The `SMTP_USER` stuff is contamination from the `opencart` changes. Not related to this change.

Reply:
true they were needed for testing

Reply:
I added fixes for the SMTP settings in https://github.com/ksylvan/docker-mail-server/pull/10